core/lib/multivm/src/versions/vm_latest/tests/nonce_holder.rs

```diff
@@ -101,7 +102,7 @@ fn test_nonce_holder() {
     run_nonce_test(
         1u32,
         NonceHolderTestMode::SetValueUnderNonce,
-        Some("Previous nonce has not been used".to_string()),
+        Some("Error function_selector = 0x13595475, data = 0x13595475".to_string()),
```

Comment:
it would be really nice to have some programmatic way to generate those. these errors wont be readable by someone who does not know that to search for preimage of the selector you need to do full search over the repo.

also, it is not convenient to generate those test cases

Reply:
thats a good point, ill see what we can do